wheel_diameter : 0.34
wheel_base: 1.070
steering_track: 1.0
wheel_steering_y_offset: 0.0
imu_topic_name : "/small_scout_2/imu_filtered"
odometry_out_topic_name : "/small_scout_2/dead_reckoning/odometry"
joint_state_topic_name : "/small_scout_2/joint_states"
odometry_frame_id : "small_scout_2_odom"
odometry_child_frame_id : "small_scout_2_base_footprint"
joint_state_frame_id : "small_scout_2_chassis"
